domains/transfer/Transfer.API/API/ApiModels/Requests/RuleExtensions.cs

```diff
@@ -13,4 +14,25 @@ public static class RuleExtensions
         => ruleBuilder
             .LessThan(253402300800)
             .WithMessage("{PropertyName} too high! Please make sure the format is UTC in seconds.");
+
+    public static IRuleBuilderOptions<T, string> MustBeValidBase64EncodedWalletDepositEndpoint<T>(this IRuleBuilder<T, string> ruleBuilder)
```

Comment:
There does not appear to be any wallet / deposit / endpoint checking in this rule, perhaps it should just be named `MustBeValidBase64`?

... Is there perhaps a builtin rule/validation for simple base64?

Reply:
I can't seem to find a built in rule in fluent validation for base64, so I will stick to using the TryFromBase64String method, you suggested, in your other comment